app/controllers/api/v1x0/mixins/index_mixin.rb

```diff
@@ -32,10 +32,6 @@ def rbac_scope(relation)
           end
         end
 
-        def pagination_params
```

Comment:
@syncrou @eclarizio 
We need to get rid of this because it has all kinds of issues when we set 
ActionController::Parameters.action_on_unpermitted_parameters = :raise

and we see all kinds of parameters getting trapped because of this code.